src/components/menus/workMenu.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="ts">
+  import MenuContents from '@src/components/menuContents.svelte';
+  import IconButton from '@src/components/iconButton.svelte';
+  import WorkMenuEntry from '@src/components/workMenuEntry.svelte';
+  import { workStores, saveResumeDataToLocalStorage, WorkStore } from '@src/data/data';
+  import { arrayMove } from '@src/util/arrayMove';
+  import { get, type Writable } from 'svelte/store';
+  let work: Writable<WorkStore[]> = workStores;
```

Comment:
Same thing here, can I just make the assignment without importing stuff I don't need?